src/Commands/Local/CloneCommand.php

```diff
@@ -37,19 +37,22 @@ class CloneCommand extends TerminusCommand implements SiteAwareInterface, Config
      *
      * @option site_dir Custom directory for the local copy. By default, the site name is used
      * @option override Override the local copy if exists
+     * @option branch The branch to clone. Default is master
      *
      * @usage <site> Clone's a local copy into "$HOME/pantheon-local-copies"
      */
-    public function clone(string $site_id, array $options = ['site_dir' => null, 'override' => null]): string
+    public function clone(string $site_id, array $options = ['site_dir' => null, 'override' => null, 'branch' => null]): string
```

Comment:
Hi!
I think it's better if you put the default value 'master' in the function args and then $devBranch is always just the value of $options['branch']

Could you please make this small change? I could then test this real quick and get it merged :)